locale/en.po

```diff
@@ -1017,161 +1017,239 @@ msgstr "Cilia"
+msgstr "Run a certain number of worlds, the amount of generations for each world to run is determined by the above spin box."
+
+#: ../src/auto-evo/AutoEvoExploringTool.cs:643
 msgid "FINISH_X_GENERATIONS"
 msgstr "Finish {0} Generations"
 
-#: ../src/auto-evo/AutoEvoExploringTool.tscn:105
+#: ../src/auto-evo/AutoEvoExploringTool.cs:659
+msgid "RUN_X_WORLDS"
+msgstr "Run {0} Worlds"
+
+#: ../src/auto-evo/AutoEvoExploringTool.cs:783
+#: ../src/auto-evo/AutoEvoExploringTool.tscn:340
+msgid "READY"
+msgstr "Ready"
+
+#: ../src/auto-evo/AutoEvoExploringTool.cs:933
+msgid "CURRENT_WORLD_STATISTICS"
+msgstr ""
+"[b]Generations:[/b]\n"
+"  {0}\n"
+"[b]Total Patches:[/b]\n"
+"  {1}\n"
+"[b]Total Auto-Evo Time:[/b]\n"
+"  {2}\n"
+"[b]Total Species:[/b]\n"
+"  {3}\n"
+"[b]Species Still Alive:[/b]\n"
+"  {4}\n"
+"[b]Species Count per Patch:[/b]\n"
+"  Average {5}; Standard deviation {6}\n"
+"[b]Total Population:[/b]\n"
+"  {7}\n"
+"[b]Most Populous Species:[/b]\n"
+"  {8}\n"
+"[b]Basic Microbe Species Data:[/b]\n"
+"  Average hex size: {9}\n"
+"[b]Generic Organelle Data:[/b]\n"
+
+#: ../src/auto-evo/AutoEvoExploringTool.cs:947
+#: ../src/auto-evo/AutoEvoExploringTool.cs:991
+msgid "MICROBE_ORGANELLE_STATISTICS"
+msgstr "  {0}: Found in {1} species, averaging {2} each\n"
+
+#: ../src/auto-evo/AutoEvoExploringTool.cs:975
+msgid "ALL_WORLDS_STATISTICS"
+msgstr ""
+"[b]Generations:[/b]\n"
+"  {0}\n"
+"[b]Total Species:[/b]\n"
+"  Average {1}; Standard deviation {2}\n"
+"[b]Species Still Alive:[/b]\n"
+"  Average {3}; Standard deviation {4}\n"
+"[b]Per-patch Species Count:[/b]\n"
+"  Average {5}; Standard deviation {6}\n"
+"[b]Per-patch Total Population:[/b]\n"
+"  Average {7}; Standard deviation {8}\n"
+"[b]Microbe Species Average Hex Size:[/b]\n"
+"  Average {9}; Standard deviation {10}\n"
+"[b]Generic Organelle Data[/b]\n"
+
+#: ../src/auto-evo/AutoEvoExploringTool.tscn:114
+msgid "WORLD"
+msgstr "World"
+
+#: ../src/auto-evo/AutoEvoExploringTool.tscn:122
 msgid "CONFIG"
 msgstr "Config"
 
-#: ../src/auto-evo/AutoEvoExploringTool.tscn:113
+#: ../src/auto-evo/AutoEvoExploringTool.tscn:130
 msgid "MAP"
 msgstr "Map"
 
-#: ../src/auto-evo/AutoEvoExploringTool.tscn:121
+#: ../src/auto-evo/AutoEvoExploringTool.tscn:138
 #: ../src/microbe_stage/editor/MicrobeEditorTabButtons.tscn:113
 msgid "REPORT"
 msgstr "Report"
 
-#: ../src/auto-evo/AutoEvoExploringTool.tscn:129
+#: ../src/auto-evo/AutoEvoExploringTool.tscn:146
 msgid "VIEWER"
 msgstr "Viewer"
 
-#: ../src/auto-evo/AutoEvoExploringTool.tscn:177
+#: ../src/auto-evo/AutoEvoExploringTool.tscn:182
+msgid "ALL_WORLDS_GENERAL_STATISTICS"
+msgstr "Generic Statistics of All Worlds"
+
+#: ../src/auto-evo/AutoEvoExploringTool.tscn:226
+msgid "WORLD_GENERAL_STATISTICS"
+msgstr "Generic Statistics of Current World"
+
+#: ../src/auto-evo/AutoEvoExploringTool.tscn:269
+msgid "CURRENT_WORLD"
+msgstr "Current World"
+
+#: ../src/auto-evo/AutoEvoExploringTool.tscn:289
+msgid "INIT_NEW_WORLD_TOOLTIP"
+msgstr "Init a new world"
```

Comment:
I think this should use the world "create" instead of init as this is user readable text (initialize could be an alternative but I think create would be clearer to players).

This is the last thing I want to comment on.